src/status_im/ui/screens/wallet/send/events.cljs

```diff
@@ -265,4 +277,4 @@
 (handlers/register-handler-fx
   :wallet.send/set-gas-price
   (fn [{:keys [db]} [_ gas-price]]
-    {:db (assoc-in db [:wallet :send-transaction :gas-price] (money/bignumber gas-price))}))
\ No newline at end of file
+    {:db (assoc-in db [:wallet :send-transaction :gas-price] (money/bignumber gas-price))}))
```

Comment:
Can you exclude those changes from your commits? Looks like it's back and forth and sometimes causes git to notice changes on my env.

Reply:
having newline at the end of files is the correct convention, if your editor removes it configure it.

Reply:
The correct convention is the one we agree on :)
I have a similar issue with `package-lock.json` and I don't think I touch this file at all.